salt/modules/zypper.py

```diff
@@ -58,65 +61,223 @@ def __virtual__():
     return __virtualname__
 
 
-def _zypper(*opts):
-    '''
-    Return zypper command with default options as a list.
-
-    opts
-        additional options for zypper command
-
-    '''
-    cmd = ['zypper', '--non-interactive']
-    cmd.extend(opts)
-
-    return cmd
-
-
-def _is_zypper_error(retcode):
-    '''
-    Return True in case the exist code indicate a zypper errror.
-    Otherwise False
-    '''
-    # see man zypper for existing exit codes
-    return int(retcode) not in [0, 100, 101, 102, 103]
+class _Zypper(object):
```

Comment:
Wouldn't it be better to implement this class in salt/utils/pkg/ and call it only Zypper() (without the underscore)?
Would make this file much smaller.